.eslintignore

```diff
@@ -3,3 +3,4 @@
 /assets/*
 /temp*
 /repositories/*
+/lib/substitute.js
```

Comment:
If we're going to be maintaining this code from within metalsmith-permalinks itself going forward, I see no reason why we should exempt it from the usual lint rules. (Though if you'd rather address that in a later commit I'd be okay with that too.)